pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -740,7 +740,9 @@ func buildSidecarOutboundTCPListenerOptsForPortOrUDS(listenerMapKey *string,
 				destinationCIDR = svcListenAddress
 				listenerOpts.bind = actualWildcard
 			}
-			listenerOpts.extraBind = svcExtraListenAddress
+			if len(svcExtraListenAddresses) > 0 {
```

Comment:
this `if` is not relevant? the code is the same either way

Reply:
Hi @howardjohn, I changed the  code in GetExtraAddressesForProxy to make it be covered by dual stack flag, so `len(svcExtraListenAddresses)` should be equal 0 if the dual stack is not enabled.